gabarit/template_nlp/nlp_project/tests/test_model_embedding_cnn.py

```diff
@@ -331,7 +345,7 @@ def test06_model_embedding_cnn_save(self):
         model = ModelEmbeddingCnn(model_dir=model_dir, batch_size=8, epochs=2, multi_label=False,
                                   max_sequence_length=max_sequence_length, max_words=100,
                                   padding=padding, truncating=truncating, tokenizer_filters=tokenizer_filters,
-                                  embedding_name='fake_embedding.pkl')
+                                  embedding_name='fake_embedding.pkl', random_seed=42)
```

Comment:
The second time, you should test the random_seed equal to None to vary the tests